documentation/examples/prometheus-kubernetes.yml

```diff
@@ -83,6 +82,42 @@ scrape_configs:
     target_label: __metrics_path__
     replacement: /api/v1/nodes/${1}/proxy/metrics
 
+# Scrape config for Kubelet cAdvisor.
+#
+# In Kubernetes 1.7.0 and later, cAdvisor metrics (those whose names begin with
+# 'container_') have been removed from the Kubelet metrics endpoint.  This job
+# scrapes the cAdvisor endpoint to retrieve those metrics.  This should be
+# removed in Kubernetes 1.6 and earlier versions since it will cause the metrics
+# to be scraped twice.
+- job_name: 'kubernetes-cadvisor'
+
+  # Default to scraping over https. If required, just disable this or change to
+  # `http`.
+  scheme: https
+
+  # This TLS & bearer token file config is used to connect to the actual scrape
+  # endpoints for cluster components. This is separate to discovery auth
+  # configuration because discovery & scraping are two separate concerns in
+  # Prometheus. The discovery auth config is automatic if Prometheus runs inside
+  # the cluster. Otherwise, more config options have to be provided within the
+  # <kubernetes_sd_config>.
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+
+  kubernetes_sd_configs:
+  - role: node
+
+  relabel_configs:
+  - action: labelmap
+    regex: __meta_kubernetes_node_label_(.+)
+  - target_label: __address__
+    replacement: kubernetes.default.svc:443
+  - source_labels: [__meta_kubernetes_node_name]
+    regex: (.+)
+    target_label: __metrics_path__
+    replacement: /api/v1/proxy/nodes/${1}:4194/metrics
```

Comment:
Hmm, I'm getting:

![cadvisor_error](https://user-images.githubusercontent.com/538008/28035419-d9f78514-65b4-11e7-886f-ac20e2350343.png)

...but also for the original `/api/v1/proxy/nodes/${1}:4194/metrics` here.

This is on a test cluster where I set up permissive bindings via:

```
kubectl create clusterrolebinding permissive-binding \
  --clusterrole=cluster-admin \
  --user=admin \
  --user=kubelet \
  --group=system:serviceaccounts
```

My nodes:

```
$ kubectl get nodes
NAME          STATUS    AGE       VERSION
julius-test   Ready     7d        v1.7.0
```

My Kubernetes versions:

```
Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.0", GitCommit:"d3ada0119e776222f11ec7945e6d860061339aad", GitTreeState:"clean", BuildDate:"2017-06-29T23:15:59Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.0", GitCommit:"d3ada0119e776222f11ec7945e6d860061339aad", GitTreeState:"clean", BuildDate:"2017-06-29T22:55:19Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```

I wonder what I'm missing?